src/main/java/com/google/opensesame/servlets/ProjectPreviewsServlet.java

```diff
@@ -0,0 +1,32 @@
+package com.google.opensesame.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import com.google.opensesame.projects.ProjectPreview;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/project-previews")
+public class ProjectPreviewsServlet extends HttpServlet {
+
+  private final Gson gson = new Gson();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Entity testEntity = new Entity("Project");
+    testEntity.setProperty("repositoryName", "tensorflow/tensorflow");
+    testEntity.setProperty("numMentors", 5);
```

Comment:
The test data should follow the same format as the data that comes from the API. I'm assuming that the data from the API it will be in list format, so your hardcoded data should also be in a list. This way you can write the logic to iterate over each item in the list before you try integrating the API into your code.

Reply:
That's a good point. Just to make sure I'm interpreting this correctly, you're saying that what comes from a datastore query is going to be in the format of a List of entities, and that I should be adding mock data in that format. That makes sense, and I'll do that shortly.

In case I'm misinterpreting what you're saying lmk or I'd be happy to jump on a quick call to sort it out. And in case I'm not, here's some context about what's happening here: 
The entity `testEntity` is intended to emulate an entity coming from the AppEngine datastore, where we will be storing "Projects". These project entities are where platform-specific data will be stored, such as the associated mentors. The only GitHub data stored in this entity is the name of the GitHub repository, from which we use [this object oriented representation of the GitHub API](https://github-api.kohsuke.org/) to get the additional data about the project from GitHub, as can be seen in `ProjectPreview.fromPropertiesAndRepository`.

I'm also now realizing that we haven't added our plans about how we're structuring our data to the design doc, so I'll be adding that shortly.